models/spike_detector.h

```diff
@@ -103,18 +103,12 @@ class spike_detector : public Node
   spike_detector();
   spike_detector( const spike_detector& );
 
-  void set_has_proxies( const bool hp );
   bool
   has_proxies() const
   {
     return has_proxies_;
   }
-  bool
-  potential_global_receiver() const
-  {
-    return true;
-  }
-  void set_local_receiver( const bool lr );
+
   bool
   local_receiver() const
```

Comment:
You are right. And it should return `true` -- no member` local_receiver_` required as it can no longer be set.